tests/parsers/test_xspectra.py

```diff
@@ -7,7 +7,17 @@
 
 def generate_inputs():
     """Return only those inputs that the parser will expect to be there."""
-    return AttributeDict()
+
+    parameters = {
+        'INPUT_XSPECTRA': {
+            'xepsilon(1)': 1,
+            'xepsilon(2)': 0,
+            'xepsilon(3)': 0,
```

Comment:
Are these keys now guaranteed to be in the parameters always? Because if not, the newly added line 95 in the parser will throw a `KeyError` if one of these is not defined.